IDENTITY_USE_CASES.md

```diff
@@ -0,0 +1,98 @@
+# Identity Use Cases in OpenSearch
+- [Identity Use Cases in OpenSearch](#identity-use-cases-in-opensearch)
+  - [Non-use compatibility](#non-use-compatibility)
+    - [Scenario 1:](#scenario-1)
+  - [Identity features enabled](#identity-features-enabled)
+    - [Scenario 2:](#scenario-2)
+    - [Scenario 3:](#scenario-3)
+  - [Using Admin account](#using-admin-account)
+    - [Scenario 4:](#scenario-4)
+    - [Scenario 5:](#scenario-5)
+  - [Password changes](#password-changes)
+    - [Scenario 6:](#scenario-6)
+    - [Scenario 7:](#scenario-7)
+    - [Scenario 8:](#scenario-8)
+  - [Authenticate as new user](#authenticate-as-new-user)
+    - [Scenario 9:](#scenario-9)
+    - [Scenario 10:](#scenario-10)
+    - [Scenario 11:](#scenario-11)
+  - [Multi-node Cluster](#multi-node-cluster)
+    - [Scenario XXX:](#scenario-xxx)
+
+## Non-use compatibility
```

Comment:
For my understanding, what does `non-use compatibility` mean?

Reply:
I can fill this in a little more - what happens if these feature launches and you say 'I don't want to use it'.  We should allow those customers to not be impacted by not engaging with identity features.  Note; they might be unable to use new features that take advantage - e.g. extensions so we will have ways to encourage adoption.